src/rubyLsp.ts

```diff
@@ -32,6 +34,8 @@ export class RubyLsp {
       this.currentActiveWorkspace.bind(this),
     );
     this.debug = new Debugger(context, this.workspaceResolver.bind(this));
+    // eslint-disable-next-line no-new
+    new DependenciesTree(context);
```

Comment:
We still need to figure out how to properly construct and then dispose of this object.

As things stand right now, we can't assign this to any variable, since we have no use for the variable itself (i.e. we don't do anything with it, it is just a view).

But then ESLint shouts at us if we just create an object and don't hold on to it (which is also reasonable).

I think the correct thing to do is to make `DependenciesTree` disposable, and then dispose of it when `RubyLsp` gets disposed. The problem is `RubyLsp` is itself not disposable yet.